### DISCUSSION TITLE
Bug Fixed: Now Able to Pass 'screen #num' as the Source for check_source Function

### DISCUSSION THREAD

**Discussion by n-aghaebrahim** [ultralytics/ultralytics#4155] [General]
A glitch exists within the check_source function of the build.py file. When utilizing "screen" as the source, it successfully captures the screenshot and enables model prediction. However, when specifying "screen 2" along with the screen number, it triggers an error. To fix this issue, the bug can be addressed by modifying line 120 in the build.py, as indicated in the accompanying screenshot, to:

`screenshot = True if 'screen' in source.lower() else False`

<img width="934" alt="Screenshot 2023-08-03 at 5 51 11 PM" src="https://github.com/ultralytics/ultralytics/assets/124855329/43f1cdba-c852-4364-8bd8-98f5dbc4ae00">

**Comment by glenn-jocher** [comment]
@n-aghaebrahim thank you for reporting this issue and suggesting a fix for the `check_source` function in the `build.py` file. Your contribution is greatly appreciated, and it's through community engagement like this that we can continue to improve YOLOv8.

I'll review the proposed change to ensure it aligns with the rest of the codebase and doesn't introduce any regressions. If everything checks out, we'll work on integrating the fix into an upcoming release. In the meantime, if you have any further insights or encounter additional issues, please don't hesitate to share them.

Keep an eye on the repository for updates, and thanks again for your support! 🚀
